src/lib.rs

```diff
@@ -839,25 +840,57 @@ mod group_create_opts {
     }
 }
 
+mod policy_caching_config {
+    use super::*;
+    pub fn create(max_entries: usize) -> PolicyCachingConfig {
+        PolicyCachingConfig { max_entries }
+    }
+}
+
+mod ironoxide_config {
+    use super::*;
+    pub fn create(
+        policy_caching: &PolicyCachingConfig,
+        sdk_operation_timeout: Option<&Duration>,
```

Comment:
All of the functions I made in this PR that take Options take options of borrows and use `.cloned()`. My next PR will be going through and fixing this in all our other functions.

Reply:
Since `Duration` is `Copy`, probably better to not pass a reference, especially since we have to clone it anyway...

Reply:
I didn't know that `Duration` implemented `Copy`, so that's good to know. I tried what you suggested and removed the reference, and wrote a test to see if I could re-use the `Duration`, but it failed. 
If you want to check my work, here's the test:
```
  "test" should {
    "fail" in {
      val jwt = JwtHelper.generateValidJwt(primaryTestUserId.getId)
      val timeout = Duration.from_millis(5)
      val resp1 = Try(IronOxide.userVerify(jwt, timeout)).toEither
      val resp2 = Try(IronOxide.userVerify(jwt, timeout)).toEither

      resp1.isLeft shouldBe true
      resp2.isLeft shouldBe true
    }
  }
```
I'm using the same timeout for both calls, and the first call times-out successfully, but the second call does not. So I think it's still passing a `null` after using it once. So I don't think that I should be taking a non-reference.

But knowing that `Duration` implements `Copy`, I can change my `.cloned()` calls to `.copied()`, so that's an improvement (that I'll do tomorrow).

Reply:
I guess that probably makes sense as the rust compiler is inserting the copy, but when it crosses the boundary it probably just gives it away and it gets dropped (and set to null in Java-land?)